prometheus/scripts/test-alerts.sh

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/env sh
+
+URL="http://localhost:9093/api/v1/alerts"
```

Comment:
We don't seem to use this anywhere, is this just for locally testing to see if the alert instance is up?

Reply:
Yes, that's correct. I'll get rid of this script.